org.opentosca.bus/org.opentosca.bus.management.api.java/src/main/java/org/opentosca/bus/management/api/java/MBJavaApi.java

```diff
@@ -538,9 +538,9 @@ private SituationRepository getSituationRepository() {
                 LOG.debug("instance api: {}", str);
                 map.put(para, str);
             } else if (para.equalsIgnoreCase("csarEntrypoint")) {
-                LOG.debug("Found csarEntrypoint Element! Put in instanceDataAPIUrl \""
-                    + Settings.CONTAINER_API_LEGACY + "/" + csarID + "\".");
-                map.put(para, Settings.CONTAINER_API_LEGACY + "/CSARs/" + csarID);
+            	LOG.debug("Found csarEntrypoint Element! Put in instanceDataAPIUrl \""
+                    + Settings.OPENTOSCA_CONTAINER_CONTENT_API.replace("{csarid}", csarID.csarName()));                                              
+                map.put(para, Settings.OPENTOSCA_CONTAINER_CONTENT_API.replace("{csarid}", csarID.csarName()).replace("{artifactreference}",""));
```

Comment:
*.replace("{artifactreference}",""));*  seems to be superfluous if using OPENTOSCA_CONTAINER_CONTENT_API and not OPENTOSCA_CONTAINER_CONTENT_API_ARTIFACTREFERENCE?

Reply:
removed